### DISCUSSION TITLE
mini buffer height changes constantly

### DISCUSSION THREAD

**Discussion by CDitzel** [joaotavora/eglot#817] [General]
moving point over certain names and symbols usually brings up the minibuffer with information. The height, however, does change from symbol to symbol which makes it visually unpleasent and confusing for the eye. 

Is there a workaround for this? maybe setting the minibuffer height to a fixed number in this mode?

**Comment by Eason0210** [comment]
May be  this could help.
```emacs-lisp
 (setq eldoc-echo-area-use-multiline-p nil)
```

**Reply by CDitzel** [reply]
thanks but this sets it to one line basically. Is there any other option?

**Reply by Eason0210** [reply]
may be you can try this:
```emacs-lisp
(setq eldoc-area-use-multiline-p 3) 
```
